Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -97,11 +95,35 @@ class SentryUserFeedbackForm: UIViewController {
         addedScreenshot(image: image)
         return
 #else
-        let imagePickerController = UIImagePickerController()
-        imagePickerController.delegate = self
-        imagePickerController.sourceType = .photoLibrary
-        imagePickerController.allowsEditing = true
-        present(imagePickerController, animated: config.animations)
+        
+        func presentPicker() {
+            let imagePickerController = UIImagePickerController()
+            imagePickerController.delegate = self
+            imagePickerController.sourceType = .photoLibrary
+            imagePickerController.allowsEditing = true
+            DispatchQueue.main.async {
+                self.present(imagePickerController, animated: self.config.animations)
+            }
+        }
+        
+        let status = PHPhotoLibrary.authorizationStatus()
+        switch status {
+        case .notDetermined:
```

Comment:
`l`: We should also add more detailed logging for different [status](https://developer.apple.com/documentation/photos/phauthorizationstatus#Status-Types), i.e. add a debug log when the user access is restricted, explaining SDK users that the app users have to change the setting in the system settings to enable it again.

Maybe we can also add the authorization status as context to the feedback report, so that when a developer receives a report via Sentry without an image, they can tell if the user has not selected one or if the SDK simply couldn't ask for one.

Reply:
Also doing the logging in the separate PR for photos improvement: #4723 

I'm not sure how much value there'd be in sending the photos authorization status to sentry, if there's a structured way to do that, or indeed, if it's even wise to do so from a privacy perspective.